frame/system/src/lib.rs

```diff
@@ -539,6 +572,11 @@ decl_module! {
 		}
 
 		/// Kill some items from storage.
+		/// 
+		/// # <weight>
+		/// - `O(VK)` where `V` length of `keys` and `K` length of one key
+		/// - `V` storage deletions (codec `O(K)`).
```

Comment:
I think there is no codec involved with storage deletion since the value is never read, just removed.